website/src/components/sections/Testimonials/data.js

```diff
@@ -0,0 +1,32 @@
+export const testimonials = [
+  {
+    quote: "ZIO is a great library that brings the best of functional programming to Scala. It's well designed, well documented, and has a great community.",
```

Comment:
Odersky's tweet you link to doesn't say that.
Screenshot of the tweet:
<img width="603" alt="image" src="https://github.com/user-attachments/assets/75729c75-fc32-4ffb-905b-d9440fece8b5" />

Can you please provide the link of the tweet where Odersky said that, please? or update the quote to reflect what Odersky said